extensions/systemd/systemd.js

```diff
@@ -47,6 +47,8 @@ class SystemdProcessManager extends ProcessManager {
     stop() {
         this._precheck();
 
+        this.instance.config.set('bootstrap-socket', null);
```

Comment:
This won't persist changes, you need to run the `save` method as well 😄

Reply:
good catch, ty for your input. I feel like I should "delete" it once the promise for the `systemctrl stop` command has been resolved, what do you think?

```
        return this.ui.sudo(`systemctl stop ${this.systemdName}`)
            .then((response) => {
                this.instance.config.set('bootstrap-socket', null);
                return this.instance.config.save();
            })
            .catch((error) => {
                throw new ProcessError(error);
            });
```

Reply:
In terms of when the changes to the config should be reverted, I think it would be better to do this when the [socket](https://github.com/TryGhost/Ghost-CLI/blob/1d5dab17c23541eecfd5285c2da73a48fe4380e4/lib/utils/port-polling.js) is destroyed, because it's part of the cleanup and can help mitigate issues arising from `ghost stop` not being called. However, this might be a more relevant implementation for #787

Reply:
I agree with @vikaspotluri123 here - the original idea for #805 was to clean it up after ghost was started, which would mean adding this line in an extra `then` block [here](https://github.com/TryGhost/Ghost-CLI/blob/478277f641ff264ed23e53d3962ab9b14be5a1db/extensions/systemd/systemd.js#L37) and [here](https://github.com/TryGhost/Ghost-CLI/blob/478277f641ff264ed23e53d3962ab9b14be5a1db/extensions/systemd/systemd.js#L72).

Also, `config.set()` returns the config instance, so you can just use `config.set('bootstrap-socket', null).save();` 😄

Reply:
@acburdine ty for your input as well, I've refactorized the code to clean it up after ghost was started (start and restart commands). I kept the original `this.instance.config` call to make it consistent with the other portions where config is called in the file.